pkg/migrate/migrate.go

```diff
@@ -427,7 +429,11 @@ func getPVCs(ctx context.Context, w *log.Logger, clientset k8sclient.Interface,
 			if err != nil {
 				return nil, nil, fmt.Errorf("failed to get PVC for PV %s in %s: %w", pv.Spec.ClaimRef.Name, pv.Spec.ClaimRef.Namespace, err)
 			}
-			matchingPVCs[pv.Spec.ClaimRef.Namespace] = append(matchingPVCs[pv.Spec.ClaimRef.Namespace], *pvc)
+
+			if (pv.Spec.ClaimRef.Namespace == Namespace && Namespace != "") || Namespace == "" {
```

Comment:
```suggestion
			if pv.Spec.ClaimRef.Namespace == Namespace ||Namespace == "" {
```
this is logically equivalent and a lot cleaner